### ISSUE TITLE
pana should enable super mixins over flutter packages

### ISSUE THREAD

**Issue by kevmoo** [dart-lang/pub-dartlang-dart#829]
URL: https://pub.dartlang.org/packages/pdrpulm#-analysis-tab-

Pretty easy

```yaml
analyzer:
  strong-mode: true
  language:
    enableSuperMixins: true
```

**Comment by kevmoo** [comment]
Not sure if we should do all of these for flutter - 

https://github.com/flutter/flutter/blob/37f216bfabfb2cba3a882c819e373b141a1aac89/packages/flutter/lib/analysis_options_user.yaml#L22

**Comment by kevmoo** [comment]
We could just *use* this file straight from the cloned flutter SDK, right @isoos ?

**Comment by isoos** [comment]
Supermixin is not available on the web, so the setting mustn't be global. If the package is known to be flutter-only, we could use the stricter superset of the settings, and also enable the supermixin.

**Comment by kevmoo** [comment]
Will be fixed when we land pana 0.8.1 - https://github.com/dart-lang/pub-dartlang-dart/issues/833

**Comment by kevmoo** [comment]
This is fixed in latest analysis! https://analyzer-dot-dartlang-pub.appspot.com/packages/pdrpulm/0.0.2/5649050225344512
